erlang.mk

```diff
@@ -596,7 +596,7 @@ pkg_check_node_commit = master
 
 PACKAGES += chronos
 pkg_chronos_name = chronos
-pkg_chronos_description = Timer module for Erlang that makes it easy to abstact time out of the tests.
+pkg_chronos_description = Timer module for Erlang that makes it easy to abstract time out of the tests.
```

Comment:
I have PRs to the upstreams for these changes, so while accepting this change now would make the code out of sync with the upstream, it shouldn't remain that way for particularly long